packages/frontend/src/Components/AddTutorBox/style.scss

```diff
@@ -1,35 +1,35 @@
 .add-tutor-box-component {
-    margin: 1em auto;
-    float: right;
+  margin: 1em auto;
+  float: right;
 }
 
 .add-tutor-box-component p {
-    margin: 0.25em auto;
-    text-align: center;
+  margin: 0.25em auto;
+  text-align: center;
 }
 
 .add-tutor-box-button {
-    background-color: #FF5E84;
-    border: #FF5E84 1px solid;
-    color: white;
-    box-sizing: border-box;
-    cursor: pointer;
-    text-transform: uppercase;
+  background-color: #ff5e84;
```

Comment:
Lowercase hexidecimal is less readable than uppercase IMO

Reply:
Understood. If you have any other suggestions. Please let me know. 😄

Reply:
https://github.com/prettier/prettier/issues/5158
Seems they are not going to support that.